haskell/haskell.bzl

```diff
@@ -130,13 +130,21 @@ def _mk_binary_rule(**kwargs):
         ),
     )
 
-    # Tests have an extra fields regarding expected code coverage percentages.
+    # Tests have an extra fields regarding code coverage.
     if is_test:
         attrs.update({
-            "expected_expression_coverage": attr.int(
-                default = 0,
+            "expected_covered_expressions_percentage": attr.int(
+                default = -1,
```

Comment:
Is this `-1` documented?

Reply:
Actually I don't like sentinel values, but the bazel API does not allows us to use something similar to a `Maybe`. However any sentinel value must be documented as much as possible, OR using something more common such as `None` if possible.

Reply:
Added a comment internally, but I don't think this should be documented to the user. This is the Nothing value, and if the user provides anything it should be a Just.